src/styles/filter.scss

```diff
@@ -0,0 +1,22 @@
+.filter {
+  display: flex;
+
+  &__link {
+    margin: 3px;
+    padding: 3px 7px;
+
+    color: inherit;
+    text-decoration: none;
+
+    border: 1px solid transparent;
+    border-radius: 3px;
+
+    &:hover {
+      border-color: rgba(175, 47, 47, 0.1);
```

Comment:
Create mixin for this color (pass opacity to the params).

[More info.](https://sass-lang.com/documentation/at-rules/mixin/)

Reply:
Try to use HEX instead of RGBA. This is a more concise version

Reply:
The opacity property makes the whole element (including any text) semi-transparent, so I used an RGBA format

Reply:
![image](https://github.com/mate-academy/react_todo-app/assets/96265844/88c07051-4e58-4d77-a227-cce8ce45fe64)